proto/autoops/service.proto

```diff
@@ -160,6 +161,67 @@ message ListWebhooksResponse {
   int64 total_count = 3;
 }
 
+message CreateProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  CreateProgressiveRolloutCommand command = 2;
+}
+
+message CreateProgressiveRolloutResponse {}
+
+message GetProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+}
+
+message GetProgressiveRolloutResponse {
+  ProgressiveRollout progressive_rollout = 1;
+}
+
+message DeleteProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
```

Comment:
To keep the consistency, can you set the `environment_namespace` as the first property in the message?
Same for the other messages.